generators/server/templates/src/test/java/package/web/rest/LogoutResourceIT.java.ejs

```diff
@@ -91,14 +81,12 @@ public class LogoutResourceIT {
     private WebTestClient webTestClient;
 <%_ } _%>
 
-    private OidcIdToken idToken;
```

Comment:
idToken is used in getLogoutInformation method

Reply:
yes thanks